WooCommerce/Classes/ViewRelated/Dashboard/TopPerformers/TopPerformersDashboardView.swift

```diff
@@ -0,0 +1,145 @@
+import SwiftUI
+import enum Yosemite.StatsTimeRangeV4
+
+/// SwiftUI view for the Top Performers dashboard card.
+///
+struct TopPerformersDashboardView: View {
+    @ObservedObject private var viewModel: TopPerformersDashboardViewModel
+    @State private var showingCustomRangePicker = false
+
+    private let onViewAllAnalytics: (_ siteID: Int64,
+                                     _ timeZone: TimeZone,
+                                     _ timeRange: StatsTimeRangeV4) -> Void
+
+    init(viewModel: TopPerformersDashboardViewModel,
+         onViewAllAnalytics: @escaping (Int64, TimeZone, StatsTimeRangeV4) -> Void) {
+        self.viewModel = viewModel
+        self.onViewAllAnalytics = onViewAllAnalytics
+    }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            header
+                .padding(.horizontal, Layout.padding)
+                .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                .shimmering(active: viewModel.syncingData)
+
+            timeRangeBar
+                .padding(.horizontal, Layout.padding)
+                .redacted(reason: viewModel.syncingData ? [.placeholder] : [])
+                .shimmering(active: viewModel.syncingData)
+
+            Divider()
+
+            // TODO
+
+            Divider()
```

Comment:
Can be added in this PR or in the next one, just a reminder that this divider needs a bit of left spacing according to Figma:

<img width="369" alt="Screenshot 2024-04-18 at 14 16 31" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/21116479-57ec-4993-abd7-354c1f92a8e9">

Reply:
Also updated the padding in 5357cbcf683c6d91eec46bfac09d85ce9ef73603.